src/qml/DockPage.qml

```diff
@@ -112,6 +112,55 @@ ItemPage {
             Item {
                 Layout.fillHeight: true
             }
+            
+            Label {
+                text: qsTr("Dock Transparency")
+                color: Meui.Theme.disabledTextColor
+                bottomPadding: Meui.Units.smallSpacing
+            }
+
+            TabBar {
+                id: transparencyTabBar
+                Layout.fillWidth: true
+
+                TabButton {
+                    text: qsTr("Enabled")
+                }
+
+                TabButton {
+                    text: qsTr("Disabled")
+                }
```

Comment:
could you make this a switch? https://doc.qt.io/qt-5/qml-qtquick-controls2-switch.html

Reply:
doing it rn

Reply:
@omaemae done